docs/api-reference/react/deckgl.md

```diff
@@ -13,56 +13,44 @@ The `DeckGL` class is a React wrapper of the `Deck` JavaScript class which expos
 // Basic usage
 import DeckGL, {ScatterplotLayer} from 'deck.gl';
 
-const App = (viewState, data) => (
+const App = (data) => (
   <DeckGL
-    {...viewState}
+    longitude={-122.45}
+    latitude={37.78}
+    zoom={12}
     layers={[new ScatterplotLayer({data})]} />
 );
 ```
 
-Like any React component, `DeckGL` can accept child components. Child components can be automatically positioned and sized to fit perfectly underneath any `View` in the list of `views`. Child components are often maps (e.g. the `StaticMap` component from react-map-gl), but can be any React components.
+Like any React component, `DeckGL` can accept child components. Child components are often maps (e.g. the `StaticMap` component from react-map-gl), but can be any React components.
 
 ```js
-import DeckGL, {MapView, FirstPersonView} from 'deck.gl';
-// Multiple views and an auto-positioned base map
-const views = [
-  new FirstPersonView({...}),
-  new MapView({id: 'basemap', ...})
-];
-
-render() {
-  return (
-    <DeckGL
-      width={width}
-      height={height}
-      layers={this._renderLayers()}
-      views={views} />
-
-      <StaticMap
-        viewId='basemap'
-        {...viewState}/>
-
-    </DeckGL>
-  );
-}
+import DeckGL, {MapController} from 'deck.gl';
+import {StaticMap} from 'react-map-gl';
+
+const App = (data) => (
+  <DeckGL
+    initialViewState={{longitude: -122.45, latitude: 37.78, zoom: 12}}
+    controller={MapController}
+    layers={[new ScatterplotLayer({data})]}
+  >
+    {({width, height, viewState}) => (
```

Comment:
Should we have both an example that uses the function style, and one that does not, and comment on how they relate?